### PR TITLE
fix: add values for etcd helm chart

### PR DESCRIPTION
**Pull request by waveywaves**
- refactor helm value types 
- etcd helm chart values

### CODE REVIEW

File: controllers/constants/constants.go [hunk 1]
```diff
@@ -19,4 +19,13 @@ const (
 	VCLUSTER_K8S_DISTRO         = "k8s"
 	K3S_DATASTORE_ENDPOINT      = "K3S_DATASTORE_ENDPOINT"
 	VCLUSTER_INGRESS_HOSTNAME   = "VCLUSTER_INGRESS_HOST"
+	DEFAULT_K3S_VERSION         = "rancher/k3s:v1.27.3-k3s1"
+	UCLUSTER_SYNC_PLUGIN_TAG    = "uffizzi/ucluster-sync-plugin:v0.2.4"
+	OCI_TYPE                    = "oci"
+	PREMIUM_RWO_STORAGE_CLASS   = "premium-rwo"
+	NO_SCHEDULE                 = "NoSchedule"
```
Comment (axisofentropy) [inline controllers/constants/constants.go:26]: This is available within the k8s.core.v1 module as [v1](https://pkg.go.dev/k8s.io/api/core/v1).[TaintEffectNoSchedule](https://pkg.go.dev/k8s.io/api/core/v1#TaintEffectNoSchedule)

but I don't know if that module is available to everywhere that needs it below

File: controllers/constants/constants.go [hunk 2]
```diff
@@ -19,4 +19,13 @@ const (
...
+	SANDBOX_GKE_IO_RUNTIME      = "cloud.google.com/gke-sandbox"
+	EXISTS                      = "Exists"
```
Comment (axisofentropy) [inline controllers/constants/constants.go:28]: probably available as `v1.NodeSelectorOpExists`

Comment (waveywaves) [reply controllers/constants/constants.go:28]: it is available, I am using it now

File: controllers/constants/constants.go (hunk 1 above)
Comment (waveywaves) [reply controllers/constants/constants.go:26]: it is available, I am using it now

**Pull request merged**